pages/support/ticket_sidebar.vue

```diff
@@ -2,6 +2,7 @@
   <div v-if="!$fetchState.pending">
     <user-details
       :user-id="requester.externalId"
+      :order-id="(orderId) ? orderId : null"
```

Comment:
this can be dropped

Reply:
Can this still be dropped since we need it in the component?

I do think we could simplify this though:

```js
 :order-id="orderId"
```

If it isn't set it will already be `null` so no need for the ternary expression.

Reply:
```suggestion
      :order-id="orderId"
```